app/views/users/index.html.erb

```diff
@@ -1,16 +1,13 @@
-<h1>Index (Will load users in this page)</h1>
 
+    <% @users.each do |user| %>
+        <div class="all-user-container">
+        <div class="user-img-container">
+        <img src="<%= user.photo %>" />
+        </div>
+        <div class="user-details-container">
+        <h2><%= user.name %></h2>
+        <h3>Number of Posts: <%= user.postscounter %></h3>
+        </div>
+        </div>
```

Comment:
- [ ] Clicking on each user should take you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)